applications/sasquatch/charts/app-metrics/templates/telegraf-configmap.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sasquatch-telegraf-app-metrics
+  labels:
+    app.kubernetes.io/name: sasquatch-telegraf-app-metrics
+    app.kubernetes.io/instance: sasquatch-telegraf-app-metrics
+    app.kubernetes.io/part-of: sasquatch
+data:
+  telegraf.conf: |+
+    [agent]
+      metric_batch_size = 5000
+      metric_buffer_limit = 100000
+      collection_jitter = "0s"
+      flush_interval = "10s"
+      flush_jitter = "0s"
+      debug = {{ default false .Values.debug }}
+      omit_hostname = true
+
+    [[outputs.influxdb]]
+      urls = [
+        {{ .Values.influxdb.url | quote }}
+      ]
+      database = "telegraf-kafka-app-metrics-consumer"
+      username = "${INFLUXDB_USER}"
+      password = "${INFLUXDB_PASSWORD}"
+
+    [[outputs.influxdb]]
+      namepass = ["telegraf_*"]
+      urls = [
+        {{ .Values.influxdb.url | quote }}
+      ]
+      database = "telegraf"
+      username = "${INFLUXDB_USER}"
+      password = "${INFLUXDB_PASSWORD}"
+
+    {{- range $index, $app := .Values.apps }}
+    [[inputs.kafka_consumer]]
+      brokers = [
+        "sasquatch-kafka-brokers.sasquatch:9092"
+      ]
+      consumer_group = "telegraf-kafka-consumer-app-metrics"
+      sasl_mechanism = "SCRAM-SHA-512"
+      sasl_password = "$TELEGRAF_PASSWORD"
+      sasl_username = "telegraf"
+      data_format = "avro"
+      avro_schema_registry = "http://sasquatch-schema-registry.sasquatch:8081"
+      avro_timestamp = "timestamp_ns"
+      avro_timestamp_format = "unix_ns"
+      avro_union_mode = "nullable"
+      avro_tags = {{ include "helpers.toTomlArray" (index $.Values.globalAppConfig $app "influxTags")  }}
```

Comment:
You'll probably always have at least one tag configured for the app, but I would be a little more defensive here testing if `influxTags` is not empty first.